main.go

```diff
@@ -86,6 +86,11 @@ func init() {
 func main() {
 	ctx := context.Background()
 	oktetoLog.Init(logrus.WarnLevel)
+	if registrytoken.IsRegistryCredentialHelperCommand(os.Args) {
```

Comment:
@jLopezbarb just implemented a new type of logger "silent" in order to control stdout... maybe his implementation can be used here too? https://github.com/okteto/okteto/pull/3941